app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,51 @@ def update
         end
       end
 
+      def exist
+        userData = params[:user]
+        Rails.logger.info "************ exist is = #{userData[:email]}"
+        user = User.where(email: userData[:email]).first
+        Rails.logger.info "************ user is = #{user}"
+        @redirect_path = '/profile' #change this to actually redirect to the social login function
+        unless user
+             @redirect_path = '/additional_info'
```

Comment:
Here too.

Also, can you outdent 2 spaces.

Reply:
outdenting is done.

Reply:
For the redirect_path see above.